src/client/components/draft/rich_text/rich_text.tsx

```diff
@@ -123,11 +123,14 @@ export class RichText extends Component<Props, State> {
 
     const contentBlocks = convertHtmlToDraft(
       html,
-      hasLinks,
+      hasLinks || false,
```

Comment:
Should this logic exist in the constructor for this class so that you won't have to remember to  say`hasLinks || false` every time you use it?

Reply:
Good call, this was unnecessary!